src/import/import.rs

```diff
@@ -258,7 +255,7 @@ impl<Client: ImportClient> ImportSSTJob<Client> {
 
     fn run(&mut self) -> Result<()> {
         let start = Instant::now();
-        info!("{} start {:?}", self.tag, self.sst);
+        info!("start"; "tag" => %self.tag, "sst" => ?self.sst);
```

Comment:
em, do we have a guide that when to use `%`, and when to use `?`?
@breeswish @overvenus

Reply:
`%` is for showing use `Display`, `?` is for showing use `Debug` (just like `{}` for `Display` and `{:?}` for `Debug`). We don't have a standard that for what we should use debug and for what we should use display. But *personally* I think all of these things to be outputted should implement `Display` instead of `Debug` (using `Debug` is ususally an abuse)